std/conv.d

```diff
@@ -3872,8 +3872,18 @@ private S textImpl(S, U...)(U args)
 
         auto app = appender!S();
 
+        // assume that on average, parameters will have less
+        // than 20 elements
+        app.reserve(U.length * 20);
+
         foreach (arg; args)
-            app.put(to!S(arg));
+        {
+            static if (is(Unqual!T == uint) || is(Unqual!T == ulong) || is(Unqual!T == int) || is(Unqual!T == long))
```

Comment:
Ah, I see. I wonder why `toChars` defines its constraint to only take u/ints and u/longs... Maybe I'll take a look sometime and see if it can be expanded/simplified.